comms/EclCC.cpp

```diff
@@ -725,11 +725,26 @@ boost::filesystem::path EclCCSubPath(const boost::filesystem::path & directory)
     return directory / "clienttools" / "bin" / "eclcc.exe";
 }
 
-unsigned int FindAllCEclCC(const boost::filesystem::path & progFiles, CEclCCVector & results)
+void PushUniqueEclCC(CEclCCVector& results, boost::filesystem::path eclccPath)
 {
-    if (boost::filesystem::exists(progFiles)) {
+    bool found = false;
+    for (auto& c : results)
```

Comment:
Should be using `std::find`

Reply:
I used "find_if" given it is a comparison of a value in the object.